tests/lib/wast/Cargo.toml

```diff
@@ -19,6 +19,7 @@ wast = "38.0"
 serde = "1"
 tempfile = "3"
 thiserror = "1.0"
+tokio = { version = "1", features = [ "io-util", "rt" ], default_features = false }
```

Comment:
Why is tokio needed for tests? Can we use other task manager

Reply:
Because the crate contains wasi tests, and our wasi implementation now requires tokio.

Reply:
@syrusakbary did that clear it up?
We basically have tokio as a hard dependency for WASI now.

They only way to not have that dependency would be to split out WASI tests into a separate crate.

Reply:
Assumed as answered - it's required.